Brain/Brain.ino

```diff
@@ -126,8 +126,8 @@ void loop() {
 void sensorReadFunc(int id) {
   switch (id) {
     case 0:
-      Thermocouple::setSensor(0);
-      Thermocouple::readTemperatureData(farrbconvert.sensorReadings);
+      //Thermocouple::setSensor(0);
+      Ducers::readTemperatureData(farrbconvert.sensorReadings);
```

Comment:
I believe for case 1 and 2, you need to include the farrbconvert

Reply:
Also I think it would make sense to move sensorReadFun to brain utils rather than cluttering the brain code

Reply:
will move func to brain utils;
But what do you mean by the first statement?

Reply:
Look at the lines below. For case 0, once you got the data you did: 
farrbconvert.sensorReadings[1] = tempController::controlTemp(farrbconvert.sensorReadings[0]);
farrbconvert.sensorReadings[2] = -1;

Don't you need to do this for case 1 and case 2 or is it already doing that when you pass in farrbconvert.sensorReadings as a parameter

Reply:
It's done in the client side read functions.

Reply:
I see sounds good